packages/ui/src/text-field/TextFieldOverrides.tsx

```diff
@@ -29,11 +32,40 @@ export function overrideTextField(theme: SuperDispatchTheme): void {
   theme.overrides.MuiInputBase = {
     root: {
       ...theme.typography.body2,
-      '&$disabled': { backgroundColor: Color.Silver100 },
+      '&$disabled': {
+        backgroundColor: Color.Silver30,
+      },
+      '&$error': { borderColor: Color.Red500 },
```

Comment:
@umrzoq-toshkentov Shouldn't it be ":" instead of "$"? Or is "$" something specific too material ui?

Reply:
@timurhamitov I also initially thought like that and it is specific for material-ui `makeStyles`  only. I think it is for type of mode(disabeld, error and etc.). It helps us to easily update style